source/plugin/text/pagination.rst

```diff
@@ -3,16 +3,16 @@ The Pagination Service
 ======================
 
 .. javadoc-import::
+    net.kyori.adventure.audience.Audience
     org.spongepowered.api.service.pagination.PaginationList.Builder
     org.spongepowered.api.service.pagination.PaginationService
-    org.spongepowered.api.text.channel.MessageReceiver
 
 .. tip::
 
     For a basic understanding of services, make sure you read :doc:`../services` first.
 
 The :javadoc:`PaginationService` acts as a way to split up content into discrete pages. The service provides a
-:javadoc:`PaginationList.Builder` with which you can specify attributes such as title, contents, header, and padding.
+:javadoc:`PaginationList.Builder` with which you can specify attributes such as title, contents, header, and padding. This is a Sponge-specific feature.
```

Comment:
New line here please, per our guidelines (max 120 characters per line).

Reply:
Do you have a quick way for me to lint my `.rst` files? Would be nice to have to find all of the line length problems together.

Reply:
I came up with a way and found 1029 lines across the entire docs source tree. I think I'll just fix the ones from my diff :sweat_smile:

Reply:
I think we give a bit of leeway when it contains a URL. It's mainly for readability when reviewing ...